backend/organization/views/organization_views.py

```diff
@@ -378,6 +400,40 @@ def post(self, request):
             )
 
 
+class LookUpOrganizationAPIView(APIView):
+    def get(self, *args, **kwargs):
+        query = self.request.query_params.get("search")
+        if Organization.objects.filter(name__iexact=query).exists():
+
+            organization = Organization.objects.filter(name__iexact=query)
+
+            return Response(
+                {
+                    "message": "Organization with name {} already exists".format(query),
+                    "url": organization[0].url_slug,
+                },
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+        if OrganizationTranslation.objects.filter(
+            name_translation__iexact=query
+        ).exists():
+            organization_translation = OrganizationTranslation.objects.filter(
+                name_translation__iexact=query
+            )
+            return Response(
+                {
+                    "message": "Organization with name {} already exists".format(
+                        query
+                    ),
+                    "url": organization_translation[0].organization.url_slug,
+                },
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+        return Response(
+            status=status.HTTP_200_OK
+        )
```

Comment:
@JoshKornfeld why do you need to create an API for looking up existing organization?

Reply:
The previous implementation didn't handle case sensitivity, or check if the name existed as a translation for the name of another org. Since the previous implementation already makes an API call, I thought it could be suitable to make an API call that specifically does all the checks we need.

Reply:
Okay, that makes sense!